wandb/__init__.py

```diff
@@ -846,6 +846,8 @@ def init(job_type=None, dir=None, config=None, project=None, entity=None, reinit
     Args:
         job_type (str, optional): The type of job running, defaults to 'train'
         config (dict, argparse, or tf.FLAGS, optional): The hyper parameters to store with the run
+        config_exclude_keys (list, optional): List of keys to exclude when specifying config
+        config_include_keys (list, optional): List of keys to include when specifying config
```

Comment:
We should make this more clear as to what it's excluding or including.  Maybe exclude from storing in W&B?

Reply:
good idea. thanks, fixed.